go/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:16.04
```

Comment:
This `Dockerfile` is used for `PFSServer`, so we can move this to  `./cmd/pfsserver` ?